<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirWindow</class>
 <widget class="QMainWindow" name="DirWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DirWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="IconView" name="listView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QColumnView" name="columnView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>608</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionIcon_View"/>
    <addaction name="actionList_View"/>
    <addaction name="separator"/>
    <addaction name="actionReorder_Icons"/>
   </widget>
   <addaction name="menuDisplay"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUp"/>
   <addaction name="actionDebug"/>
   <addaction name="actionReorder_Icons"/>
   <addaction name="actionIcon_View"/>
   <addaction name="actionList_View"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUp">
   <property name="text">
    <string>Up</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionReorder_Icons">
   <property name="text">
    <string>Reorder Icons</string>
   </property>
  </action>
  <action name="actionIcon_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Icon View</string>
   </property>
  </action>
  <action name="actionList_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>List View</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>IconView</class>
   <extends>QListView</extends>
   <header>iconview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
